src/tools/tidy/src/deps.rs

```diff
@@ -81,12 +81,10 @@ const EXCEPTIONS: ExceptionList = &[
     ("ar_archive_writer", "Apache-2.0 WITH LLVM-exception"), // rustc
     ("colored", "MPL-2.0"),                                  // rustfmt
     ("dissimilar", "Apache-2.0"),                            // rustdoc, rustc_lexer (few tests) via expect-test, (dev deps)
-    ("encoding_rs", "(Apache-2.0 OR MIT) AND BSD-3-Clause"), // opt-dist
     ("fluent-langneg", "Apache-2.0"),                        // rustc (fluent translations)
     ("fortanix-sgx-abi", "MPL-2.0"),                         // libstd but only for `sgx` target. FIXME: this dependency violates the documentation comment above.
     ("instant", "BSD-3-Clause"),                             // rustc_driver/tracing-subscriber/parking_lot
     ("mdbook", "MPL-2.0"),                                   // mdbook
-    ("openssl", "Apache-2.0"),                               // opt-dist
```

Comment:
Why are we no longer tracking the deps for opt-dist? I see that there's a bunch of dependencies that disappeared from our lockfile as well -- where are those being locked? I don't see a new Cargo.lock added here...

Reply:
We were able to remove those dependencies from `opt-dist`, since it no longer needs to fetch anything over the network.

I removed those lines because, IIRC, `tidy` complained that they were no longer reflected in the Cargo.lock once I removed the deps from `opt-dist`, which made sense.

Reply:
Ah, perfect. I missed that part of the diff in my skim I guess :)

Cutting some dependencies is great.